bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -26,25 +26,21 @@ f = h5py.File(args.output_file, "w")
 logging.info('Copying segments and attributes to %s' % args.output_file)
 # Move segments information into the final file - remove some duplication in earlier files
 for fi in files:
-    # Get a list of ifos in the segments of the input data file
-    ifos_in_file = [ifo for ifo in args.ifos if ifo in fi['segments']]
-    # Arbitrarily choose one to copy over the segments for as all are currently the same 
-    ifo_to_get = 'segments/{}'.format(ifos_in_file[0])
-    # Convert list of IFOs into a identifier string
-    ifosstring = ''.join(ifos_in_file)
-    group_to_save = '{}/segments'.format(ifosstring)
-    f[group_to_save + '/end'] = fi[ifo_to_get + '/end'][:]
-    f[group_to_save + '/start'] = fi[ifo_to_get + '/start'][:]
-    # Use this ID string to save the attributes from the input files
+    if len(fi['segments'].keys()) > 1:
+        logging.info('Incorrect segment format.')
+
+    key = fi['segments'].keys()[0]
+    f['segments/%s/end' % key] = fi['segments/%s/end' % key][:]
+    f['segments/%s/start' % key] = fi['segments/%s/start' % key][:]
     for attr_name in fi.attrs:
-        f[ifosstring].attrs[attr_name] = fi.attrs[attr_name]
+        f['segments/%s' % key].attrs[attr_name] = fi.attrs[attr_name]
 
```

Comment:
As we are creating a coinc group for attributes above, then we should do the same here have a `key` group in this file as well (i.e. `f[key].attrs[attr_name]`)

Reply:
We are not creating it, it already exists, the segments are saved in there for the specific combination, hence there is no point in creating a new group.

Reply:
Sorry I wasn't clear, I was suggesting that we _do_ create a `f[key]` group, even though `f['segments/%s' % key]` exists, for the attributes which are not related to the segments

Reply:
ah there is an error in here, as in there can be a segments/foreground from coinc_statmap. This needs to change for the same check as it is later in hdfinjfind. Though the key is the detector combination, so I am not sure what attribute you want to save not specific to detector combination, key-ed by detector combination

Reply:
It should be correct now.

Reply:
so the pivot and fixed ifos are not segment related, but are combination related, so my suggestion is to have a different group, just called `f[key]` and not have the 'segments/'

Reply:
changed as requested